docs/tutorials/ssl.md

```diff
@@ -1,6 +1,6 @@
-# SSL Connections
+# TLS/SSL Connections
 
-Mongoose supports connecting to [MongoDB clusters that require SSL connections](https://www.mongodb.com/docs/manual/tutorial/configure-ssl/). Setting the `ssl` option to `true` in [`mongoose.connect()`](../api/mongoose.html#mongoose_Mongoose-connect) or your connection string is enough to connect to a MongoDB cluster using SSL:
+Mongoose supports connecting to [MongoDB clusters that require TLS/SSL connections](https://www.mongodb.com/docs/manual/tutorial/configure-ssl/). Setting the either the `tls` or `ssl` option to `true` in [`mongoose.connect()`](../api/mongoose.html#mongoose_Mongoose-connect) or your connection string is enough to connect to a MongoDB cluster using TLS/SSL:
 
 ```javascript
 mongoose.connect('mongodb://127.0.0.1:27017/test', { ssl: true });
```

Comment:
We should recommend `tls: true` instead of `ssl: true`. The `ssl` option looks like it is still supported by MongoDB Node driver, but given that all other `ssl*` options have been removed we should recommend using `tls: true`

Reply:
Thanks for reviewing. Great suggestion. I have updated this to include only `tls: true`.